libs/application/templates/financial-aid/src/forms/MuncipalityNotRegistered.ts

```diff
@@ -1,28 +1,15 @@
-import {
-  buildCustomField,
-  buildForm,
-  buildSection,
-  Form,
-  FormModes,
-} from '@island.is/application/core'
+import { buildCustomField, buildForm, Form } from '@island.is/application/core'
 
 import * as m from '../lib/messages'
 
 export const MuncipalityNotRegistered: Form = buildForm({
   id: 'FinancialAidApplication',
-  title: m.application.name,
-  mode: FormModes.REJECTED,
+  title: '',
```

Comment:
why are we removing this?

Reply:
This is to remove everything from the sidebar:
<img width="468" alt="image" src="https://user-images.githubusercontent.com/43727721/171424918-1d6ac563-4340-433a-b077-de1937165a75.png">